mypy/message_registry.py

```diff
@@ -80,6 +80,8 @@ def with_additional_msg(self, info: str) -> ErrorMessage:
 ASYNC_FOR_OUTSIDE_COROUTINE: Final = '"async for" outside async function'
 ASYNC_WITH_OUTSIDE_COROUTINE: Final = '"async with" outside async function'
 
+AWAIT_WITH_OUTSIDE_COROUTINE: Final = '"await" outside coroutine ("async def")'
```

Comment:
Did you really intend to call it `AWAIT_WITH_OUTSIDE_COROUTINE`? This looks like a copying issue to me, why `WITH`? And please remove the blank line before this defn, those three messages are related to each other

Reply:
indeed was a yanking error. thank you for pointing that out.